articles/140_topic_and_service_name_mapping.md

```diff
@@ -246,13 +309,18 @@ Here is a non-exhaustive list of prefixes:
 While all planned prefixes consist of two characters, i.e. `rX`, anything proceeding the first namespace separator, i.e. `__`, can be considered part of the prefix.
 The standard reserves the right to use up to 8 characters for the prefix in case additional prefix space is needed in the future.
 
-### Communicating with Non-ROS Topics
+#### Communicating with Non-ROS Topics
 
 Since all ROS topics are prefixed when being converted to DDS topic names, it makes it impossible to subscribe to existing DDS topics which do not follow the same naming pattern.
 For example, if an existing DDS program is publishing on the `image` topic (and is using the DDS equivalent to the ROS message type) then a ROS program could not subscribe to it because of the name mangling.
 Therefore to allow ROS programs to interoperate with "native" DDS topic names the API should provide a way to skip the ROS specific prefixing.
 
-### ROS to DDS Name Conversion Examples
+#### Substitution of the Namespace Delimiter
+
+The namespace delimiter in ROS 2 topic and service names, a forward slash (`/`), will be replaced with double underscores (`__`).
```

Comment:
At the top of the document you removed the constraints about parts not being allowed to start / end / contain underscores. If the forward slashes here are still replaced with double underscores that constrain can't go away. Otherwise the resulting string would be ambiguous. Can the partition not contain forward slashed according to the standard? Or does it allow them?